docs/zh/deployment/distributed-deployment-guide.md

```diff
@@ -512,6 +512,28 @@ spring.cloud.consul.host=127.0.0.1
 spring.cloud.consul.port=8500
 ```
 
+##### 2.2.1.2.9 启用外部Zookeeper服务注册中心替换内置eureka
+1. 修改build.sh/build.bat，将`config-service`和`admin-service`的maven编译命令更改为
+```shell
+mvn clean package -Pgithub -DskipTests -pl apollo-configservice,apollo-adminservice -am -Dapollo_profile=github,zookeeper-discovery -Dspring_datasource_url=$apollo_config_db_url -Dspring_datasource_username=$apollo_config_db_username -Dspring_datasource_password=$apollo_config_db_password
+```
+2. 分别修改apollo-configservice和apollo-adminservice安装包中config目录下的application-github.properties，配置zookeeper服务器地址
+```properties
+spring.cloud.zookeeper.connect-string=127.0.0.1:2181
+```
+3.本地调试需要分别修改`apollo-configservice`和`apollo-adminservice`服务对应的JVM启动参数
+```properties
+-Dapollo_profile=github,zookeeper-discovery
```

Comment:
Is it better not to include the local development instructions? Because this is the distributed deployment guide, most of the readers read this guide to deploy in a production environment. The local development instructions here might confuse them.